apps/web/components/ui/UsernameAvailability/UsernameTextfield.tsx

```diff
@@ -116,9 +131,7 @@ const UsernameTextfield = (props: ICustomUsernameProps) => {
             ref={usernameRef}
             name="username"
             value={inputUsernameValue}
-            addOnLeading={
-              <>{process.env.NEXT_PUBLIC_WEBSITE_URL.replace("https://", "").replace("http://", "")}/</>
-            }
+            addOnLeading={<>{usernamePrefix}/</>}
```

Comment:
Can't we just pass `addOnLeading` in the parent? So `UsernameTextfield` doesn't care about org-specific logic.